packages/create-remix/templates/express/server/index.js

```diff
@@ -46,3 +43,11 @@ function purgeRequireCache() {
     }
   }
 }
+
+let remixBuildPath = path.join(process.cwd(), 'public', 'build');
```

Comment:
As of #809, we're using `const` in the docs

```suggestion
const remixBuildPath = path.join(process.cwd(), 'public', 'build');
```

Reply:
updated.